src/app/contextBroker/contextBroker.cpp

```diff
@@ -265,6 +265,8 @@ bool            statTiming;
 bool            statNotifQueue;
```

Comment:
CNR entry needed, fe:

"Add: id fields checking for NGSIv2 API and (if -ngsiv1CheckIdFields is enabled) NGSIv1 API (#1601)